application-xpoll-ui/src/main/resources/XPoll/XPollJSExtension.xml

```diff
@@ -170,6 +170,22 @@
       var hiddenField = '#option' + $(this).val();
       $(hiddenField).val(proposal);
     });
+    // Sticky header for Internet Explorer 9+
+    var val = $('.xpoll th:lt(1)');
```

Comment:
Isn't this the same as:

```suggestion
    var val = $('.xpoll th:first');
```

Reply:
As I also mentioned in the comments added in the last commit, the problem was that IE didn't support `display: sticky;`. To fix this, I wrapped the table in a container and checked if the display property was set or not on the table headers. If it was not set, it meant that we were running in IE. I set the container height to be that of the viewport and made it scrollable. Lastly, I added a scroll listener on that container and offset the table headers by the distance the user has scrolled.

The changes don't affect the supported browsers. This is how it looks in Microsoft Edge.
![normalBrowser](https://user-images.githubusercontent.com/45433221/149327732-6d511484-9349-4c02-9731-ff3f43dbdb9f.gif)

This is how it looks on IE.
![internet explorer](https://user-images.githubusercontent.com/45433221/149327761-842e14d6-2012-46e1-811f-5960b4685bac.gif)